app/src/main/java/com/google/samples/apps/topeka/fragment/QuizFragment.java

```diff
@@ -241,6 +242,7 @@ public void showSummary() {
         scorecardView.setAdapter(mScoreAdapter);
         scorecardView.setVisibility(View.VISIBLE);
         mQuizView.setVisibility(View.GONE);
+        Snackbar.make(mQuizView, "Congratulations! You gained " + mCategory.getScore() + " marks in this category!", Snackbar.LENGTH_SHORT).setAction("Action", null).show();
```

Comment:
You put there action button with nulled `OnClickListener`, which is definitely not right solution. I think you should (choose one of these options):
-  remove `setAction()` method which is already disused
-  add any action like in this scenario: when user click on "OK" button, Snackbar is dismissed. 

Check: https://stackoverflow.com/questions/30729312/how-to-dismiss-a-snackbar-using-its-own-action-button